### DISCUSSION TITLE
OGN-OUT

### DISCUSSION THREAD

**Discussion by gauthier108** [VirusPilot/stratux-pi4#11] [General]
Hello VirusPilot,
Do you know what to do to be able to send OGN messages via Stratus.
I notice on some pictures on your page that there is a connection for an OGN antenna that send messages.

**Comment by VirusPilot** [comment]
You need additional hardware to do it: https://github.com/b3nn0/stratux/wiki/Build-example-3:-Include-transmission-capabilities,-so-others-can-see-you-as-well. At this point, the T-Beam is the best solution.
